client/src/components/Profile/ProfileTabs.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+import BasicTabs from "components/Tabs/BasicTabs";
+import styled from "styled-components";
+
+// Styled component named StyledButton
+const StyledTabs = styled.div`
```

Comment:
I think you can do styled(BasicTabs), then you don't have to wrap this around a div

Reply:
That makes sense, but I am not able to get it to work without the div wrapper. Marking resolved for now and if it poses as a breaking feature later on I'll address then